mne/preprocessing/tests/test_maxwell.py

```diff
@@ -970,4 +970,13 @@ def test_triux():
                             st_duration=4., verbose=True)
     assert_meg_snr(sss_py, read_crop(tri_sss_st4_fname), 700., 1600)
 
+@testing.requires_testing_data
```

Comment:
try running `make flake` in the `mne-python` git root, you'll see this function should have two blank lines before it (Travis will also complain about this for us)